src/pages/FilmPage.js

```diff
@@ -59,7 +59,11 @@ class FilmPage extends Component {
   }
 
   componentDidMount() {
-    this.performSearch()
+    this.fetchData()
+  }
+
+  fetchData() {
```

Comment:
No need to create separate function for calling another function. Just call it directly